yarn-project/world-state/src/synchronizer/server_world_state_synchronizer.test.ts

```diff
@@ -49,36 +50,19 @@ const getMockContractData = () => {
   return ContractData.random();
 };
 
-const getMockGlobalVariables = () => {
-  return GlobalVariables.from({
-    chainId: Fr.random(),
-    version: Fr.random(),
-    blockNumber: Fr.random(),
-    timestamp: Fr.random(),
-  });
-};
-
 const getMockL1ToL2MessagesData = () => {
   return new Array(NUMBER_OF_L1_L2_MESSAGES_PER_ROLLUP).map(() => Fr.random());
 };
 
+// TODO: this is very similar to L2Block.random, move the functions to a shared place
```

Comment:
Is there an issue for this or should it be part of this?

Reply:
Will create the issue. I was also thinking that it's a bad practice to have a test-only relevant code in production so we should probably go through all the random() functions and see if it's used in prod and if not move it into separate mocks module. WDYT?

Reply:
How much of the code is not test-only 😝. But yes, some of these are only expected to be used in tests, but I'm seeing most of the code here as a large test and not a prod release 😬

Reply:
You have a point 😆 . Well anyway cleaned this up in https://github.com/AztecProtocol/aztec-packages/pull/3827/commits/6bed1316eebc108dc9b08c990001a5712bc5242a